node/bft/src/primary.rs

```diff
@@ -572,12 +573,12 @@ impl<N: Network> Primary<N> {
         let mut transmissions = self.sync_with_batch_header_from_peer(peer_ip, &batch_header).await?;
 
         // Check that the transmission ids match and are not fee transactions.
-        for (transmission_id, transmission) in transmissions.iter_mut() {
+        if let Err(err) = transmissions.par_iter_mut().try_for_each(|(transmission_id, transmission)| {
```

Comment:
Why not use `cfg_iter_mut`?

Reply:
we could, but do we support snarkOS without `rayon` :thinking:?

Reply:
I think it is better to use it as a form of convention

Reply:
done